README.md

```diff
@@ -1,7 +1,55 @@
 # Vanilla OS Handbook
-This is the official handbook for Vanilla OS. It contains tutorials, guides and other information to help you get the most out of your Vanilla OS installation.
+
+- This is the official handbook for Vanilla OS. It contains tutorials, guides and
+other information to help you get the most out of your Vanilla OS installation.
+- Vanilla OS Handbook utilises Jekyll for generating pages.
+- Thanks for your interest in our project. This guide is for writing and translating articles.
 
 ## Writing a guide
-If you want to write a guide for the handbook, you can do so by adding the
-new article to the `_posts` folder. The filename must be in the format
-`YYYY-MM-DD-title.md`. Then open a new pull request.
+
+- If you want to write a guide for the handbook, add the
+new article to the `_posts` directory.
+- The filename must be in the format
+`YYYY-MM-DD-title.md`. 
+- Add your GitHub username with the co-author's username in the
+author field to have a working link when the post is published. 
+- Set the layout as an
+`article`.
+- Headings must follow a proper hierarchy.
+- Refer [markdownlint](https://github.com/DavidAnson/markdownlint) to know the best practices for using markdown.
+- The guide must be structured in the following way at the beginning:-
+
+```md
+---
+title:
+description:
+date: YYYY-MM-DD
+layout: article
+authors: 
+    - 1st author GitHub username
+    - 2nd author GitHub username
+    - ...
+---
+
+Contents
```

Comment:
```suggestion
```
This can be removed too. What do you guys think?

Reply:
I don't think it's necessary